prac_05/emails.py

```diff
@@ -1,4 +1,31 @@
 """
 CP1404/CP5632 Practical
 emails
-"""
\ No newline at end of file
+"""
+
+
+def main():
+    """Create dictionary of emails to names."""
+    email_dict = {}
+    email = input("Email: ")
+    while email != "":
+        user_name = get_name(email)
+        user_confirmation = input("Is you name {}? (Y or N)".format(user_name))
+        if user_confirmation.upper() == "N" and user_confirmation != "":
```

Comment:
if: statement logic here is incorrect, the [user confirmation != ""] code will never be used, since the logic line will be False if user_confirmation != "N"

suggested line: if user_confirmation.upper() == "N"